### PR TITLE
Update aws no credential provider troubleshooting and psql SSL error

### PR DESCRIPTION
**Pull request by greedy52**

### CODE REVIEW

**Review by hugoShaka — APPROVED** [review]

File: docs/pages/includes/aws-no-credential-provider.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### No credential providers error
+
+If you see the error `NoCredentialProviders: no valid providers in chain` in {{service}} Service logs then Teleport
+is not detecting the required credentials to connect via AWS IAM permissions. Check whether
+the credentials or security role has been applied in the machine running the Teleport {{service}} Service.
+
+When running on EKS, this error may occur if the Teleport {{service}} Service
+cannot access IMDSv2 when PUT requests hop limit on the worker node instance is
```
Comment (ptgott) [inline docs/pages/includes/aws-no-credential-provider.mdx:8]:
```suggestion
cannot access IMDSv2 when the PUT requests hop limit on the worker node instance is
```

File: docs/pages/includes/aws-no-credential-provider.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+set to 1. You can check the value of `HttpPutResponseHopLimit` by running
+command [`aws ec2
```
Comment (ptgott) [inline docs/pages/includes/aws-no-credential-provider.mdx:10]: I would put example commands on their own lines using code fences to make this easier to read.

**Review by ptgott — APPROVED** [review]

File: docs/pages/includes/aws-no-credential-provider.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+the credentials or security role has been applied in the machine running the Teleport {{service}} Service.
+
+When running on EKS, this error may occur if the Teleport {{service}} Service
+cannot access IMDSv2 when the PUT requests hop limit on the worker node
+instance is set to 1. You can use the following commands to check and update
+the hop limit:
+```code
+$ aws ec2 describe-instances --instance-ids <node-instance-id> | grep HttpPutResponseHopLimit
+                        "HttpPutResponseHopLimit": 1,
+$ aws ec2 modify-instance-metadata-options --http-put-response-hop-limit 2 --instance-id <node-instance-id>
```
Comment (hugoShaka) [inline docs/pages/includes/aws-no-credential-provider.mdx:14]: I know you just got asked to put this command, but this is not the appropriate way to solve the problem.

EKS nodes are created via nodesets, they are not managed by the userdirectly. If you manually bump the hot limit on specific instances, this modification will be lost the next time a node update happens.

I think we should remove the "updater" part from the command as we have no easy way to tell the user how to fix their instance

Comment (greedy52) [reply docs/pages/includes/aws-no-credential-provider.mdx:14]: thanks for catching this. updated.

**Pull request merged**